signup-service/pom.xml

```diff
@@ -9,11 +9,11 @@
 	<parent>
 		<groupId>io.mosip.signup</groupId>
 		<artifactId>signup-parent</artifactId>
-		<version>1.1.1-SNAPSHOT</version>
+		<version>1.1.2-SNAPSHOT</version>
 	</parent>
 
 	<artifactId>signup-service</artifactId>
-	<version>1.1.1-SNAPSHOT</version>
+	<version>${project.parent.version}</version>
```

Comment:
Due to the build script requirement in this service pom.xml we cannot use ${project.parent.version}. kindly hardcode the value as 1.1.2-SNAPSHOT